types/jss/observable.d.ts

```diff
@@ -0,0 +1,17 @@
+// Copied from https://github.com/cssinjs/jss/blob/6ed7963786d5ef899075e95f81efc9530342154f/src/types.js
```

Comment:
Those were, in turn, copy/pasted from https://github.com/material-motion/indefinite-observable-js.

@DanielRosenwasser can we import these from there?  I feel like they are supposed to be whitelisted somewhere, but I can't find the whitelist (and I imagine it's less bureaucracy if someone else whitelists it, since I don't think I'm on an Microsoft NDA).

Reply:
I not 100% clear what you are asking, but I think you're asking whether `@types/jss` can reference indefinite-observable-js instead of copying its types here. To do that, 

1. Add indefinite-observable-js as a dependency to package.json
2. Change to `import { Observable } from 'indefinite-observable-js'`
3. Remove this file.

There's no whitelist that I know of. Can you explain what you mean?

Reply:
I thought there was an MS-maintained list of dependencies that an @types package could have, but I was having a hard time finding the link.

Reply:
I am not a lawyer, but I think DefinitelyTyped is legally separate from Microsoft even though the Typescript team spends a lot of time here. At any rate, typings can have whatever dependencies they want, and I think depending directly on indefinite-observable-js is a good idea.

Reply:
@sandersn: I just tried again in #24710 and found the whitelist:

> If this is an external library that provides typings,  please make a pull request to types-publisher adding it to `dependenciesWhitelist.txt`.

Reply:
I don't have time to go through the CLA process.  @sandersn, would you be able to add indefinite-observable to that whitelist, so #24710 can build correctly?